content/Phoenix for Rails developers - Bruno Vezoli.md

```diff
@@ -0,0 +1,347 @@
+# Phoenix for Rails developers: a practical example
+
+![Phoenix logo](images/phoenix-header.png)
+
+I started working on Ruby after an exploratory phase of many languages lead me to it. The syntax just captivated me, it was so expressive that it could be read like English and it had so many tools I could do things more productively than ever before. No more string helper functions or awkward loops to manipulate arrays, you could all those things out of the box and just focus on coding your solution.
```

Comment:
`and it had so many tools I could do things more productively than ever before.` This sentence is a little strange, maybe `and it had so many tools that let me do things more productively than ever before.`?